fdbserver/DataDistribution.actor.cpp

```diff
@@ -637,6 +642,12 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self,
 				TraceEvent("DataDistributionEnabled").log();
 			}
 
+			state Reference<DDTenantCache> ddtc;
```

Comment:
Nit: `ddtc` might also refer to `ddTeamCollection`. `tenantCache` might be a more clear variable name.